### ISSUE TITLE
Crashes with generator_stop

### ISSUE THREAD

**Issue by Fuyukai** [wildfoundry/dataplicity-lomond#68]
```
[2018-06-23 14:55:56.220575] ERROR: lomond: error in websocket loop
Traceback (most recent call last):
  File "/home/laura/.local/share/virtualenvs/jokusoramame-SR0dS4Ee/lib/python3.7/site-packages/lomond/stream.py", line 52, in feed
    frame = next(iter_frames)
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/laura/.local/share/virtualenvs/jokusoramame-SR0dS4Ee/lib/python3.7/site-packages/lomond/session.py", line 394, in run
    for event in self.websocket.feed(data):
  File "/home/laura/.local/share/virtualenvs/jokusoramame-SR0dS4Ee/lib/python3.7/site-packages/lomond/websocket.py", line 272, in feed
    for message in self.stream.feed(data):
RuntimeError: generator raised StopIteration
```

**Comment by willmcgugan** [comment]
I think Python 3.7 introduced some new behaviour regarding StopIteration and generators.

**Comment by willmcgugan** [comment]
Fixed in 0.3.1
